README.md

```diff
@@ -6,6 +6,8 @@ In OCDS, the signatories to the contract are not explicitly declared in the `con
 
 In some types of contracting processes, there can be additional signatories to the contract, or the signatories to the contract can differ from those specified in `buyer` (`publicAuthority`) and `suppliers` (`preferredBidders`). Use this extension only in such circumstances.
 
+When using `contracts.signatories` all signatories should be included, even the `buyer` and `suppliers`. If the `buyer` and `suppliers` are the only signatories for the contract, the `contracts.signatories` block should be omitted.
```

Comment:
I don't think this belongs in "Background". Maybe rename "Extension fields" to "Usage" and move it there? While editing that section, we can change "property" to "field". We can also merge the last sentence of the Background into Usage. So:

> Use this extension only if the signatories to a contract differ from its related award's `suppliers` and the contracting process' `buyer`. If that is the case, list all signatories to the contract, including the `buyer` and `suppliers`.